metadata/metadata.go

```diff
@@ -1900,7 +1900,8 @@ func (serv *MetadataServer) genericCreate(ctx context.Context, res Resource, ini
 	} else {
 		// TODO: create a method to get the task id from the name and variant and type
```

Comment:
Remove TODO?

Reply:
metadata and coordinator changes are done on a different PR, I will address these comments there.